script.js

```diff
@@ -0,0 +1,432 @@
+            keyboard.capsLockFunc();
+          }
+        });
+
+        key.addEventListener('mousedown', () => {
+          if (key.cl === 'ControlLeft') {
+            keyboard.controlBtnPressed = true;
+            const newEvent = new Event('customEvent');
+            window.dispatchEvent(newEvent);
+          }
+          if (key.cl === 'ShiftLeft') {
+            keyboard.capsLockFunc();
+            keyboard.shiftBtnPressed = true;
+            const newEvent = new Event('customEvent');
+            window.dispatchEvent(newEvent);
+          }
+          if (key.cl === 'ShiftRight') {
+            keyboard.capsLockFunc();
+          }
+        });
+
+        key.addEventListener('mouseup', () => {
+          if (key.cl === 'ControlLeft') {
+            keyboard.controlBtnPressed = false;
+          }
+          if (key.cl === 'ShiftLeft') {
+            keyboard.shiftBtnPressed = false;
+            keyboard.capsLockFunc();
+          }
+          if (key.cl === 'ShiftRight') {
+            keyboard.capsLockFunc();
+          }
+        });
+
+        key.addEventListener('focus', () => {
+          key.blur();
+        });
+
+        this.elements.keysContainer.appendChild(key);
+      }
+      this.elements.keysContainer.appendChild(document.createElement('br'));
+    }
+  },
+
+  notesInnerHTML: `<p class="note"><span class="note__important">IMPORTANT NOTE:</span> <span class="note__warning">Virtual Keyboard <span class="note__underline">supports AltGraph</span> button!</span> So:<br>
+  - if you have this button on your real keyboard<br>
+  (or)<br>
+  - if you press RIGHT ALT button when your OS system (!) uses layout other than English<br>
+  <span class="note__underline">Then BOTH ControlLeft and AltRight Virtual Keyboard keys will be clicked at the same time!</span><br>
+  See "AltGraph" here: <a href="https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState" target="_blank">https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState</a></p><p class="lang-key-combination">Use LeftCTRL + LeftShift to switch between languages</p>`,
+
+  init() {
+    // Create main elements
+    this.elements.input = document.createElement('input');
+    const notes = document.createElement('div');
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+
+    // Setup main elements
+    this.elements.input.classList.add('input-field');
+    notes.innerHTML = this.notesInnerHTML;
+    notes.classList.add('notes-container');
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard__keys');
+
+    // Put elements into DOM
+    document.body.appendChild(this.elements.input);
+    document.body.appendChild(notes);
+    this.elements.main.appendChild(this.elements.keysContainer);
+    document.body.appendChild(this.elements.main);
+
+    keyboard.createKeys();
+  },
+
+  capsLockFunc() {
```

Comment:
Is it OK to use such long names (consists of 5 words)? I thought that the name should consist of from 1 to 3 words maximum. So that's why I used such a name...